### ISSUE TITLE
curl-pipe-bash doesn't detect "sudo /bin/bash"

### ISSUE THREAD

**Issue by creste** [returntocorp/sgrep-rules#2786]
**Describe the bug**
Putting `sudo` in front of `/bin/bash` prevents [curl-pipe-bash](https://semgrep.dev/r?q=bash.curl.security.curl-pipe-bash.curl-pipe-bash) from finding a problem.

**To Reproduce**
Scan this bash code with the [curl-pipe-bash](https://github.com/returntocorp/semgrep-rules/blob/develop/bash/curl/security/curl-pipe-bash.yaml#L2) rule:
```bash
curl http://10.110.1.200/deployment/scripts/SLAVE00-flight-setup.bash | sudo /bin/bash
```

**Expected behavior**
I expected semgrep to find curl output being piped to bash. 

**Actual behavior**
The rule fails to find curl output being piped to bash because the rule does not allow `sudo` in front of `/bin/bash`:
https://github.com/returntocorp/semgrep-rules/blob/dd5032e036006d1cd30c3234193488feea0aa1b6/bash/curl/security/curl-pipe-bash.yaml#L30-L31

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [X] P2: annoying but not blocking me
